### DISCUSSION TITLE
How does local saving on html files in the compiler work?

### DISCUSSION THREAD

**Discussion by David-Orangemoon** [TurboWarp/packager#517] [General]
I was just wondering how that works because I was thinking of making an unsandboxed extension that lets me do the same thing but without having to have a bunch o cloud variables.

**Comment by CST1229** [comment]
I'd assume it simply uses `localStorage.setItem("varname", "value")` and `localStorage.getItem("varname")`.

**Reply by David-Orangemoon** [reply]
Thanks I will try it out!
